Converters/Revit/Speckle.Converters.RevitShared/ToHost/Raw/Geometry/CurveConverterToHost.cs

```diff
@@ -33,9 +38,34 @@ public DB.Curve Convert(SOG.Curve target)
         speckleKnots.Add(speckleKnots[^1]);
       }
 
-      //var knots = speckleKnots.GetRange(0, pts.Count + speckleCurve.degree + 1);
-      var curve = DB.NurbSpline.CreateCurve(target.degree, speckleKnots, pts, weights);
-      return curve;
+      try
+      {
+        //var knots = speckleKnots.GetRange(0, pts.Count + speckleCurve.degree + 1);
+        var curve = DB.NurbSpline.CreateCurve(target.degree, speckleKnots, pts, weights);
+        return curve;
+      }
+      // An exception was thrown by NurbSpline.CreateCurve
+      // because Revit is stricter than Rhino regarding the input parameters for NURBS curves.
+      // Exception message:
+      // "The multiplicities of other interior knots must be at most degree - 2."
```

Comment:
I would add a comment to say that this happens on semi-circle curves that were converted to nurbs